File: app/src/main/java/com/betulnecanli/mealreceipt/MealsModel.java
package com.betulnecanli.mealreceipt;



public class MealsModel {

    private String mealTitle;
    private String mealDesc;

    public MealsModel(String mealTitle, String mealDesc) {
        this.mealTitle = mealTitle;
        this.mealDesc = mealDesc;
    }

    public String getMealTitle() {
        return mealTitle;
    }

    public void setMealTitle(String mealTitle) {
        this.mealTitle = mealTitle;
    }

    public String getMealDesc() {
        return mealDesc;
    }

    public void setMealDesc(String mealDesc) {
        this.mealDesc = mealDesc;
    }
}

File: app/src/main/java/com/betulnecanli/mealreceipt/MealsModel.kt
package com.betulnecanli.mealreceipt

import com.betulnecanli.mealreceipt.MealsModel
import androidx.recyclerview.widget.RecyclerView
import com.betulnecanli.mealreceipt.MealAdapter.MealViewHolder
import android.view.LayoutInflater
import android.widget.TextView
import com.betulnecanli.mealreceipt.R
import android.view.ViewGroup
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.betulnecanli.mealreceipt.MealAdapter
import androidx.recyclerview.widget.LinearLayoutManager

data class MealsModel(var mealTitle: String, var mealDesc: String)